docs/AOTA.md

```diff
@@ -0,0 +1,43 @@
+|:----------------------|:--------------------------------|
+| Update                | Updating an application package |
+
+## Fields in the AOTA form:
+
+| Field                                             | Description                                                                                                                                                                                                                                                  |
+|:--------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| App                                               | Docker or Docker-compose or Application                                                                                                                                                                                                                      |
+| Command                                           | **Docker-Compose:** Up, Down, Pull, List and Remove.  <p>**Docker operation**s: Load, Import, Pull, Remove and Stats</p>                                                                                                                                     |
+| Container Tag                                     | Name tag for image/container.                                                                                                                                                                                                                                |
+| Docker Compose File                               | Specify custom yaml file for docker-compose command. Example: *custom.yml*                                                                                                                                                                                   |
+| Fetch                                             | Server URL to download the AOTA container *tar.gz* <p>❗ If the server requires username/password to download the file then provide this information in server username server password                                                                       |
+| Server username/<p>Server Password</p>            | If server needs credentials; specify the username and password                                                                                                                                                                                               |
+| Version                                           | Each container will have a version number tag. You are recommended to use this version number under version in the AOTA trigger. ```docker images```. See image below for result.                                                                            |
```

Comment:
There's no image below

Reply:
The version was in the developers guide, but not in the Azure or Thingsboard guide.  I couldn't find an image from where this was moved from, so I just removed that comment.